src/components/ReceiptTableCard.vue

```diff
@@ -22,16 +22,26 @@
         </template>
         <template #[`item.product`]="{ item }">
           <PriceCategoryChip v-if="item.isCategory" :priceCategory="item.category_tag" />
-          <v-text-field 
-            v-else-if="!item.productFound"
-            v-model="item.product_code"
-            :hide-details="true"
-            density="compact"
-            :rules="rules"
-            :append-inner-icon="item.product_code ? 'mdi-magnify' : 'mdi-barcode-scan'"
-            @click:append-inner="item.product_code ? findProduct(item) : launchBarcodeScanner(item)"
-            @keydown.enter="findProduct(item)"
-          />
+          <v-container v-else-if="!item.productFound">
+            <v-text-field 
+              v-model="item.product_code"
+              :hide-details="true"
+              density="compact"
+              :rules="rules"
+              :append-inner-icon="item.product_code ? 'mdi-magnify' : 'mdi-barcode-scan'"
+              @click:append-inner="item.product_code ? findProduct(item) : launchBarcodeScanner(item)"
+              @keydown.enter="findProduct(item)"
+            />
+            <div v-if="item.predicted_product_code" class="text-caption">
+              {{ $t('Common.SuggestedBarcode') }}
+              <span
+                class="fake-link"
```

Comment:
changed the css to transform to link

<img width="860" height="230" alt="image" src="https://github.com/user-attachments/assets/37eedd55-07f4-41da-882d-3151b8db0132" />